to_ass.py

```diff
@@ -197,12 +199,23 @@ def get_song_json_events(songJson, actorToStyle, shouldPrintTitle):
         start = timedelta(seconds=pytimeparse.parse(line['start']))
         end =  timedelta(seconds=pytimeparse.parse(line['end']))
 
-        romajiEvents.append(ass.line.Dialogue(
-            style=ROMAJI_STYLE_NAME,
-            start=start,
-            end=end,
-            text=get_romaji_event_text(line, actorToStyle),
-        ))
+        if (line['karaoke']):
+            romajiEvent = (ass.line.Comment(
+                style=f"Song - {songJson['title']['romaji']} {line['karaoke']}",
+                start=start,
+                end=end,
+                text=get_romaji_event_text(line, actorToStyle),
+                effect='karaoke'
+            ))
+        else:
+            romajiEvent = (ass.line.Dialogue(
+                style=ROMAJI_STYLE_NAME,
+                start=start,
+                end=end,
+                text=get_romaji_event_text(line, actorToStyle),
+            ))
```

Comment:
Seems like some unnecessary brackets in these lines

Reply:
Feels like it could just be:
```python
        if line['karaoke']:
            romajiEvent = ass.line.Comment(
                style=f"Song - {songJson['title']['romaji']} {line['karaoke']}",
                start=start,
                end=end,
                text=get_romaji_event_text(line, actorToStyle),
                effect='karaoke'
            )
        else:
            romajiEvent = ass.line.Dialogue(
                style=ROMAJI_STYLE_NAME,
                start=start,
                end=end,
                text=get_romaji_event_text(line, actorToStyle),
            )
```

Reply:
Oh, fixed